build.gradle.kts

```
@@ -3,11 +3,12 @@ import java.util.*
 plugins {
     id("java-library")
     id("maven-publish")
-    id("com.jfrog.bintray")
     id("com.github.hierynomus.license")
     id("com.github.sgtsilvio.gradle.utf8")
     id("com.github.sgtsilvio.gradle.metadata")
     id("com.github.sgtsilvio.gradle.javadoc-links")
+    id("io.github.gradle-nexus.publish-plugin")
+    signing
```

```suggestion
    id ("signing")
```
(I like consistency)